config/package/socorro/local.py

```diff
@@ -2,8 +2,8 @@
 # These settings overrides what's in settings/base.py
 
 # To extend any settings from settings/base.py here's an example:
-#from . import base
-#INSTALLED_APPS = base.INSTALLED_APPS + ['debug_toolbar']
+from . import base
+INSTALLED_APPS = base.INSTALLED_APPS + ('south',)
```

Comment:
This line shouldn't be needed. That commented out comment is out-of-date. `south` is [already in settings/base.py](https://github.com/mozilla/socorro/blob/master/webapp-django/crashstats/settings/base.py#L29)